src/pages/workspace/tags/WorkspaceViewTagsPage.tsx

```diff
@@ -231,7 +231,9 @@ function WorkspaceViewTagsPage({route}: WorkspaceViewTagsProps) {
             />
         );
     };
-
+    if (!Object.values(currentPolicyTag?.tags ?? {}).some((tag) => tag.enabled)) {
+        Tag.setPolicyTagsRequired(policyID, false, route.params.orderWeight)
+    }
```

Comment:
```suggestion

    if (!Object.values(currentPolicyTag?.tags ?? {}).some((tag) => tag.enabled)) {
        Tag.setPolicyTagsRequired(policyID, false, route.params.orderWeight);
    }

```

⚠️ @Shahidullah-Muffakir Next time after you are done with the changes, please run `npm run prettier` in order to prettify the code, otherwise the prettier related checks for the PR will fail.

In this case the `;` (semicolon) was missing at the end of the `setPolicyTagsRequired` function call.
Additionally, I added some spacing on top and bottom of the `if` for better readability.

Reply:
Got it, I will add the semicolon and adjust the spacing in this PR. I'll also make sure to run `npm run prettier `in my upcoming PRs. Thank you.